ale_linters/cs/mcsc.vim

```diff
@@ -0,0 +1,107 @@
+" general mcs options which are likely to stay constant across
+" source trees like -pkg:dotnet
+let g:ale_cs_mcsc_options = get(g:, 'ale_cs_mcsc_options', '')
```

Comment:
You might like to use `ale#Set` for these variables. Look at the codebase for other examples.